docs/build_on_linux.md

```diff
@@ -32,7 +32,7 @@ cd build
 
 After that, we're ready to build Notes!
 
-Invoke CMake to build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
+Invoke CMake to configure and build the project into a folder called `build`, in [`Release` mode](https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html):
 
 ```shell
 cmake .. -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=Release
```

Comment:
Why are we using `make -j$(nproc)` here instead of `cmake --build build` like macOS and Windows?